service-spec/schema-validator/pom.xml

```diff
@@ -61,9 +61,9 @@
                         </goals>
                         <configuration>
                             <mkdir
-                                dir="src/main/resources/hspec/${hspec.supported.version}" />
-                            <url>https://raw.githubusercontent.com/hyscale/hspec/release/HSpec${hspec.supported.version}/schema/service-spec.json</url>
-                            <outputDirectory>src/main/resources/hspec/${hspec.supported.version}</outputDirectory>
+                                dir="src/main/resources/hspec/master" />
```

Comment:
The directory should still use the variable. So that when we switch to a version, it goes to a specific directory.
We need to find a way so that if it points to master, we have one url, if it is some version, then it points to different url.

Reply:
agreed!

Reply:
But i think that part can be handled in the _**service-spec/schema-validator/src/main/resources/script/verify.sh**_ script.